lte/gateway/c/core/oai/test/CMakeLists.txt

```diff
@@ -13,6 +13,9 @@ set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+set(MAGMA_LIB_DIR $ENV{C_BUILD}/magma_common)
+link_directories(${MAGMA_LIB_DIR}/sentry)
```

Comment:
Same comment here the transitive dependencies should come from target_link_libraries pulling in the dependencies using find_packages.